config/routes.rb

```diff
@@ -52,7 +52,7 @@
     resources :sensors, only: %i[index]
 
     namespace :realtime do
-      get 'streaming_sessions' => 'sessions#index_streaming'
+      get 'streaming_sessions' => 'sessions#index_active'
```

Comment:
@AkkarinForest is that alright as for leaving a working fallback? or did you have sth else in mind when you mentioned an alias?

Reply:
Yes, that's what I had in mind. 
What do you think about creating a separate section at the bottom of this file, something like`#legacy api`? then we can also put there `get 's/:url_token'`. 

And it might be a good idea to leave a comment with the date that we change the endpoint? This way if someone works on this code and sees a very old date they will be more willing to remove it.

Reply:
added a comment but keeping it in place for now